src/Native/System.Private.CoreLib.Native/pal_dynamicload.cpp

```diff
@@ -3,9 +3,39 @@
 // See the LICENSE file in the project root for more information.
 
 #include <dlfcn.h>
+#include <string.h>
+#include "pal_common.h"
+
+#if HAVE_GNU_LIBNAMES_H
+#include <gnu/lib-names.h>
+#endif
+
+#define LIBC_NAME_WITHOUT_EXTENSION "libc"
```

Comment:
Nit: I know that the CoreCLR version has this define, but it has dubious value. I think that the code would be easier to understand if the constant is inlined in the code without this define.